docs/DeveloperGuide.md

```diff
@@ -257,14 +257,17 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* work as teaching assistants
```

Comment:
I think "works as a teaching assistant" to stay consistent with singular nouns.
Maybe could also end them with periods to stay consistent with the "Value propositin" section.

Reply:
Resolved by https://github.com/AY2223S1-CS2103T-T12-1/tp/pull/43/commits/7bf4eabce0a90a98255d040d7787420f35391459.